Sources/WishKit/SwiftUI/WishView.swift

```diff
@@ -114,6 +114,8 @@ struct WishView: View {
                         .font(.system(size: 13))
                         .multilineTextAlignment(.leading)
                         .lineLimit(descriptionLineLimit)
+                        // By changing the id, the view is identified as a new view and we avoid the weird text animation.
+                        .id(descriptionLineLimit)
```

Comment:
But don't you now have multiple views with the same ID in the hierarchy? Won't that be a problem?

Reply:
Would it be okay to generate a new `UUID(`) instead or should I create a unique id myself?

Reply:
Maybe Text + line limit? Then the ID wouldn't change every time `body` is reevaluated.